src/components/evm/UserInfo.vue

```diff
@@ -119,15 +107,12 @@ export default defineComponent({
                 >
                     <div class="c-user-info__icon-wraper"><InlineSvg
                         :src="require('src/assets/icon--acorn.svg')"
-                        :class="{
-                            'c-user-info__icon': true,
-                            'c-user-info__icon--acorn': true,
-                        }"
+                        class="c-user-info__icon c-user-info__icon--acorn"
                         height="24"
                         width="24"
                         aria-hidden="true"
                     /></div>
-                    <span class="c-user-info__menu-item-text">{{ $t('nav.teloscan') }}</span>
+                    <span class="o-text--header-5">{{ $t('nav.teloscan') }}</span>
```

Comment:
This is more like a question than a suggestion:
Do we need/like to lose an element's unique class name even when there's no actual styling need for it?

```suggestion
                    <span class="o-text--header-5 c-user-info__menu-item-text">{{ $t('nav.teloscan') }}</span>
```

It occurs to me that we do not gain anything by removing those names and yet if we keep it someone could take advantage of its presence without creating a PR to add it.

Example: it occurs to me to create a skin system that would benefit greatly if all the components are full-labeled using BEM.

What do you think

Note: this is not a blocker comment